src/js/schemas/job-schema/Schedule.js

```diff
@@ -35,7 +35,11 @@ const Schedule = {
     },
     timezone: {
       title: 'Time Zone',
-      description: 'Enter time zone in TZ format, e.g. America/New_York',
+      description: (
+        <span>
+          Enter time zone in <a href="http://www.timezoneconverter.com/cgi-bin/zonehelp" target="_blank">TZ format</a>, e.g. America/New_York.
```

Comment:
Actually, is this resource even helpful for this? I don't see TZ format described here.